lib/Embargo/Embargo.js

```diff
@@ -17,13 +18,13 @@ export default class Embargo extends React.Component {
       })
     }
 
-    renderEmbargoEnd(embargo) {
+    renderEmbargoEnd(embargo, className) {
       if (embargo.movingWallEnd?.length) {
         const unitId = `stripes-erm-components.${embargo.movingWallEnd.unit}`;
         const number = embargo.movingWallEnd.length;
         return (
           <Layout
-            className="flex centerContent"
+            className={className}
```

Comment:
you could then change this to 
```suggestion
            className={css[this.props.alignment]}
```

A new css file `Embargo.css` needs to be added for this with the following classnames

```
.left {
    display: flex; 
    align-items: center
}

.right {
   display: flex; 
    justify-content: flex-end;
}

.center {
    display: flex;
    justify-content: center
}
```

Please double check the key-values from https://github.com/folio-org/stripes-components/blob/master/lib/Layout/stories/classNamePropertiesMapping.js